src/config/validation.rs

```diff
@@ -159,7 +159,11 @@ pub fn check_outputs(config: &ConfigBuilder) -> Result<(), Vec<String>> {
     }
 
     for (key, transform) in config.transforms.iter() {
-        let outputs = transform.inner.outputs(&schema::Definition::empty());
+        let definition = schema::Definition::empty();
+        if let Err(errs) = transform.inner.validate(&definition) {
```

Comment:
Nice. Just curious as I'm not too familiar with the schema work, is there a reason `validate` needs to take a `schema::Definition` argument? It doesn't look like it's used, but I assume it's setting the stage for something.

Reply:
In some transforms (like `vrl`) we need a `schema::Definition` to generate the `outputs` and then to validate them. That's why I this field.